docs/new-project-setup.md

```diff
@@ -0,0 +1,285 @@
+# New Project Setup
+
+The DSE team regularly creates new Snowflake accounts in our Snowflake org.
+We do this instead of putting all of our data projects into our main account for a few reasons:
+
+1. At the end of a project, we often want to transfer account ownership to our partners.
+    Having it separated from the start helps that process.
+1. We frequently want to add our project champion or IT partners to our account as admins.
+    This is safer if project accounts are separate.
+1. We often want to have accounts in a specific cloud and region for compliance or data transfer regions.
+1. Different projects may require different approaches to account-level operations like OAuth/SAML.
+
+Here we document the steps to creating a new Snowflake account from scratch.
+
+## Prerequisites
+
+### Obtain permissions in Snowflake
+
+In order to create a new account, you will need access to the `orgadmin` role.
+If you have `accountadmin` in the primary Snowflake account, you can grant it to yourself:
+
+```sql
+USE ROLE accountadmin;
+GRANT ROLE orgadmin TO USER <YOUR-USER>;
```

Comment:
I think it would be a good idea to add code here for how to revoke access to the `orgadmin` role as well.

Reply:
Done